src/fides/api/alembic/privacy_experience_config_defaults.yml

```diff
@@ -0,0 +1,130 @@
+privacy_experience_configs:
+# US (non-GPP) banner and modal
+- id: pri-76b8-cc52-11ee-9eaa-8ef97a0-over-config
+  origin: pri-76b8-cc52-11ee-9eaa-8ef97a0-over
+  name: "US Modal"
+  regions:
+    - us_ca
+    - us_co
+    - us_ct
+    - us_ut
+    - us_va
+  component: modal
+  allow_language_selection: false
+  dismissable: true
+  disabled: true
+  privacy_notice_keys:
+    - data_sales_and_sharing
+
+# US (non-GPP) privacy center
+- id: pri-27d4-cc53-11ee-9eaa-8ef97a04-pri-config
+  origin: pri-27d4-cc53-11ee-9eaa-8ef97a04-pri
+  name: "US Privacy Center"
+  regions:
+    - us_ca
+    - us_co
+    - us_ct
+    - us_ut
+    - us_va
+  component: privacy_center
+  allow_language_selection: false
+  dismissable: true
+  disabled: true
+  privacy_notice_keys:
+    - data_sales_and_sharing
+
+# EU banner and modal
+- id: pri-c8ff-78d6-4a02-850f-2c09dda-over-config
+  origin: pri-c8ff-78d6-4a02-850f-2c09dda-over
+  name: "EEA Banner + Modal"
+  regions:
+    - be
+    - bg
+    - cz
+    - dk
+    - de
+    - ee
+    - ie
+    - gr
+    - es
+    - fr
+    - hr
+    - it
+    - cy
+    - lv
+    - lt
+    - lu
+    - hu
+    - mt
+    - nl
+    - at
+    - pl
+    - pt
+    - ro
+    - si
+    - sk
+    - fi
+    - se
+    - gb_eng
+    - gb_sct
+    - gb_wls
+    - gb_nir
```

Comment:
ah we need the gb reconcilation too!

Reply:
err right. i think we can just update the list here, and we shouldn't need to worry about it in terms of data migration?

Reply:
were you not looking at the regions on the privacy notices though in your migration?

Reply:
oh yeah of course, good call 😦 

i guess we can just filter out in the script, shouldn't be too bad 👍